cmd/kubeadm/app/phases/upgrade/versiongetter_test.go

```diff
@@ -0,0 +1,306 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package upgrade
+
+import (
+	"reflect"
+	"testing"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/version"
+	fakediscovery "k8s.io/client-go/discovery/fake"
+	clientsetfake "k8s.io/client-go/kubernetes/fake"
+)
+
+func Test_computeKubeletVersions(t *testing.T) {
+	tests := []struct {
+		name  string
+		nodes []v1.Node
+		want  map[string]uint16
+	}{
+		{
+			name: "kubelet version info exist",
+			nodes: []v1.Node{
+				{
+					Status: v1.NodeStatus{
+						NodeInfo: v1.NodeSystemInfo{
+							KubeletVersion: "1.28.1",
+						},
+					},
+				},
+				{
+					Status: v1.NodeStatus{
+						NodeInfo: v1.NodeSystemInfo{
+							KubeletVersion: "1.28.2",
+						},
+					},
+				},
+				{
+					Status: v1.NodeStatus{
+						NodeInfo: v1.NodeSystemInfo{
+							KubeletVersion: "1.28.2",
+						},
+					},
+				},
+			},
+			want: map[string]uint16{
+				"1.28.1": 1,
+				"1.28.2": 2,
+			},
+		},
+		{
+			name: "kubelet version info empty",
+			nodes: []v1.Node{
+				{
+					Status: v1.NodeStatus{},
+				},
+				{
+					Status: v1.NodeStatus{},
+				},
+			},
+			want: map[string]uint16{
+				"": 2,
+			},
+		},
+		{
+			name:  "node is empty",
+			nodes: []v1.Node{},
+			want:  map[string]uint16{},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := computeKubeletVersions(tt.nodes); !reflect.DeepEqual(got, tt.want) {
+				t.Errorf("got = %v, want %v", got, tt.want)
```

Comment:
```suggestion
				t.Errorf("got = %v, want = %v", got, tt.want)
```

try to be consistent.